### PR TITLE
Remove stale information about apiVersion.

### PR DESCRIPTION
**Pull request by miteshskj**
Removing stale information, referring to apiVersion change in v1.9.

### CODE REVIEW

File: content/en/docs/concepts/workloads/controllers/replicaset.md [hunk 1]
```diff
@@ -223,7 +223,7 @@ In this manner, a ReplicaSet can own a non-homogenous set of Pods
 
 As with all other Kubernetes API objects, a ReplicaSet needs the `apiVersion`, `kind`, and `metadata` fields.
 For ReplicaSets, the `kind` is always a ReplicaSet.
-In Kubernetes 1.9 the API version `apps/v1` on the ReplicaSet kind is the current version and is enabled by default. The API version `apps/v1beta2` is deprecated.
+The API version `apps/v1` on the ReplicaSet kind is the current version and is enabled by default.
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/replicaset.md:226]: This API is stable enough that you could even remove this line as well.

**Review by sftim — COMMENTED** [review]
Thanks!

File: content/en/docs/concepts/workloads/controllers/replicaset.md (hunk 1 above)
Comment (miteshskj) [reply content/en/docs/concepts/workloads/controllers/replicaset.md]: Thanks @sftim. Removed it and the subsequent line a it would be irrelevant.

**Pull request merged**